projector-plugin/src/main/kotlin/SessionDialog.kt

```diff
@@ -195,14 +196,10 @@ class SessionDialog(project: Project?) : DialogWrapper(project) {
       NetworkInterface.getNetworkInterfaces()
         .asSequence()
         .filterNotNull()
-        .filterNot { it.isLoopback }
-        .filterNot {
-          it.hardwareAddress != null
-            &&
-            it.hardwareAddress.sliceArray(0..1).contentEquals(dockerVendor)
-        }
+        .filterNot { it.isLoopback } // drop localhost
+        .filterNot { it.hardwareAddress != null && it.hardwareAddress.sliceArray(0..1).contentEquals(dockerVendor) } // drop docker
         .flatMap { it.interfaceAddresses?.asSequence()?.filterNotNull() ?: emptySequence() }
-        .mapNotNull { (it.address as? Inet4Address)?.hostName }
+        .mapNotNull { ProjectorServer.getHostName(it.address) }
```

Comment:
Maybe we should do something like `.toSet()` additionally before adding items because now I have a list containing my hostname twice.